<?xml version="1.0" ?>
<launch>
  <!-- params -->
  <param name="forest_file" value="$(find player_tracker)/config/trained_leg_detector_res=0.33.yaml" />
  <param name="scan_topic" value="/scan" />
  <param name="fixed_frame" value="/base_link"/>
  <param name="scan_frequency" value="10"/>  
  
  <!-- run detect_leg_clusters -->  
  <node pkg="player_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>
  
  <!-- run joint_player_tracker -->
  <node pkg="player_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen" />

  <!-- run joint_player_tracker -->
  <node pkg="player_tracker" type="leg_distance.py" name="leg_distance" output="screen"/>
 
   <!-- run local_occupancy_grid_mapping -->
  <node pkg="player_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/>
  
  <!-- run rviz -->
  <node pkg="rviz" type="rviz" name="rviz" />

  <!-- play rosbag normally at rate of 1.0 -->  
  <node pkg="rosbag" type="play" name="player" output="screen" required="true" 
    args="--delay=4
    --clock 
    --rate=1.0
    -s 0
    /home/airlab/Documents/bags/leg_tracker/positive_1_extracted.bag"/> <!-- test file: _2018-01-31-16-13-29.bag-->
  <param name="use_sim_time" value="true" />

  <!-- set up the cosys -->
 <node pkg="tf" type="static_transform_publisher" name="dummy_map_publisher" 
    args="0.0 0.0 0.0 0 0 0 1 map base_link 100" />
    
</launch>